BurnerB/FreeMentors-V2
# #168389920 Get all mentors

#### What does this PR do?
Allows auser to view all  mentors
#### Description of Task to be completed?
* Create the get all mentor endpoint
* Ensure tests pass
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ft-DB-get-all-mentors-168389920 
* Run npm --init to install all dependancies.
* Run command `npm test` on your terminal
* Tests should pass
* Run the command `npm start` and access g`/api/v1/mentors` :
* You should recieve a response body if the data  is present:
```
"status": 200,
    "data": [
        {
            "id": 2,
            "firstname": "John",
            "lastname": "Doe",
            "email": "johndoe6@email.com",
            "address": "Nairobi Kenya",
            "bio": "rapper, record producer, and actor who was known as one of the most-controversial and best-selling artists of the early 21st century",
            "occupation": "Musician",
            "expertise": "rapping",
            "ismentor": true,
            "isadmin": false,
            "created_on": "2019-09-10T03:25:20.407Z"
        }
    ]
```
or
```
{
    "status": 404,
    "error": "No mentors"
}
```
if no mentors are present
#### What are the relevant pivotal tracker stories?
[#168389920](https://www.pivotaltracker.com/story/show/168389920)
#### Screenshots (if appropriate) 

![get all mentors](https://user-images.githubusercontent.com/45785786/64585722-80c3f780-d3a2-11e9-8024-70351e6733aa.png)
